### ISSUE TITLE
Change color from pure black to dark grey

### ISSUE THREAD

**Issue by currandwyer** [bpmn-io/diagram-js#571]
__Is your feature request related to a problem? Please describe.__

Generally speaking, pure black #000000 should not be used in UI design. Pure black can cause eye strain and raises accessibility issues in people with dyslexia, for example.

Reference 1: [Stack Exchange discussion](https://ux.stackexchange.com/questions/23965/is-there-a-problem-with-using-black-text-on-white-backgrounds) 
Reference 2: [Never use pure black in typography](https://uxplanet.org/basicdesign-never-use-pure-black-in-typography-36138a3327a6)

__Describe the solution you'd like__

Instead of #000000 we should use the dark grey text already defined for the modeler: g-225-10-15 or #22242A

In the spirit of small incremental changes, I suggest applying this color to all elements at first - text, connectors, pools, and diagram elements.

__Describe alternatives you've considered__

A preferred option in the context of Camunda modeling software alone could be to keep the text #000000 while slightly dimming the background from pure white to for example #F8F8F8. However, this could cause issues when diagrams are exported, printed, or embedded in places outside the Camunda UI environment.

We could use different shades of grey to differentiate primary elements (like flow nodes) from secondary elements (like pools). We could also take the approach seen in certain other tools, and incorporate extraneous ui elements like gradients and drop shadows.

Clarity and simplicity is our greatest strength. We should incrementally make changes knowing that we stand on solid ground. For now additional color changes are not needed. 

__Additional context__
![Screen Shot 2021-10-06 at 10 28 15](https://user-images.githubusercontent.com/11223953/136167675-f92c874e-d5a2-4fa4-8e94-fe04acd2fd42.png)

**Comment by philippfromme** [comment]
Other tools use various colors for black.

#### Google Slides

* #000000 for title
* #595959 for subtitle
* #212121 for dark background

#### Miro

* #1a1a1a for black on the canvas
* interestingly #050038 (very dark purple) for black of the UI (icons, ...)

#### Lucidchart

* #333333 for black

Does this give us a clear direction? I don't think so. But the argument for not using actual black still stands in my opinion. Diagrams also stand out quite drastically in the Modelers as their black is much darker than the UI surrounding them.

![image](https://user-images.githubusercontent.com/7633572/169292127-114f6a15-de37-4223-8db7-a6565fd3e19c.png)

When compared with actual black only something as bright as #333333 makes a perceivable difference (#333 on the left, #000 on the right):

![image](https://user-images.githubusercontent.com/7633572/169293486-c60c69f0-1b6c-48f7-a3f3-eb7adc33f114.png)

**Comment by nikku** [comment]
What color is this: 

![image](https://user-images.githubusercontent.com/58601/169293858-c80ea224-fe00-4804-ab02-6019fe9f4ba0.png

I'd rather not go for too greyish, as it does have an impact on readability.

**Comment by philippfromme** [comment]
It's #22242a which is a dark grey with 15% luminance. #333333 when converted to HSL has 20% luminance, so it's a bit brighter.

**Comment by philippfromme** [comment]
`hsl(225, 10%, 15%)` is so close to black that people probably wouldn't even notice, which I guess would be a good thing. They might not realize that the color contrast is slightly less straining.

**Comment by philippfromme** [comment]
## Black

<img width="951" alt="black" src="https://user-images.githubusercontent.com/7633572/169329147-b1124ddb-c7c7-4db5-af07-039354c60b05.png">

## Off Black

<img width="951" alt="off-black" src="https://user-images.githubusercontent.com/7633572/169329177-3759bd66-c812-459b-9ae3-b4e3aa118ab9.png">

It's very slight but quite nice, I think. It's actually the same off black that used for the icons in the palette.

**Comment by nikku** [comment]
Just ship it and we're happier :)

**Comment by philippfromme** [comment]
Closed via https://github.com/bpmn-io/bpmn-js/pull/1656.
